packages/widget_toolkit_pin/example/lib/main.dart

```diff
@@ -111,35 +103,82 @@ class MyHomePage extends StatelessWidget {
           dialogHasBottomPadding: true,
         ),
       );
+
+  String _exampleMapMessageToString(BiometricsMessage message) {
+    switch (message) {
+      case BiometricsMessage.notSetup:
+        return 'To use biometrics, you need to turn it on in your device settings!';
+
+      case BiometricsMessage.notSupported:
+        return 'You don\'t have biometric feature on your device!';
+
+      case BiometricsMessage.enabled:
+        return 'Your biometrics are enabled!';
+
+      case BiometricsMessage.disabled:
+        return 'Your biometrics are disabled!';
+    }
+  }
 }
 
 class AppPinCodeService implements PinCodeService {
   AppPinCodeService();
 
+  final Map<String, dynamic> _data = {};
+
+  static const _isPinCodeInStorage = 'pinCode';
```

Comment:
Let's try to simplify the solution even more. As discussed the goal of the example is to have as simple as possible solution instead of a scalable solution.  I consider `_data` scalable, since it allows multiple values to be stored but we do not need this. Let's have a simple `String? _pinCode` prop. Also please add a comment that this pin is intended to be stored in the secured storage for production applications

Reply:
I have changed the `_data` to  `String? _pinCode` and added the comments.